pkg/apis/feature/features_test.go

```diff
@@ -56,10 +56,12 @@ func TestGetFlags(t *testing.T) {
 	require.True(t, flags.IsAllowed("my-enabled-flag"))
 	require.True(t, flags.IsAllowed("my-allowed-flag"))
 	require.False(t, flags.IsAllowed("non-disabled-flag"))
+
+	nodeSelector := flags.NodeSelector()
+	require.Equal(t, map[string]string{"testkey": "testvalue"}, nodeSelector)
```

Comment:
I'd like to see at least one more key/value pair here, since we are testing an implementation that allows multiple